corehq/apps/case_importer/static/case_importer/js/import_history.js

```diff
@@ -0,0 +1,43 @@
+hqDefine('case_importer/js/import_history.js', function () {
```

Comment:
Well, it's conceptually code being moved, but in order to make that happen I had to introduce a couple things like `urllib.reverse`. It would have been a cleaner diff if I'd had a commit where I just moved code but left things in a broken state, and then a second where I fixed it.